lib/guardian/plug.ex

```diff
@@ -76,6 +77,10 @@ if Code.ensure_loaded?(Plug) do
           do: GPlug.sign_in(conn, implementation(), resource, claims, opts)
 
         def sign_out(conn, opts \\ []), do: GPlug.sign_out(conn, implementation(), opts)
+
+        def remember_me(conn, mod, resource, claims \\ %{}, opts \\ []), do: GPlug.remember_me(conn, mod, resource, claims, opts)
```

Comment:
How come are `remember_me/5` and `remember_me_from_token/5` not leveraging the `implementation/1` fonction to inject the Guardian implementation module?

Reply:
I guess that would make sense. But should properly be in a new PR